onnx/defs/attr_proto_util.cc

```diff
@@ -58,4 +58,15 @@ AttributeProto MakeRefAttribute(
   return a;
```

Comment:
change this function to call the overridden one added below please.

Reply:
Oh, I think you can just merge your PR and I can rebase this branch on top of it. I remember you introduced documentation, your ordering of arguments sounds more natural to me, and I have nothing against splitting PRs in smaller piece.

Reply:
Aha, I abandoned my PR this morning (realized it's duplicate with changes in your PR). Let me get it back and merge it in this way :)